server/command.go

```diff
@@ -236,3 +237,34 @@ func (p *Plugin) runPopCommand(args []string, extra *model.CommandArgs) (bool, e
 
 	return false, nil
 }
+
+func getAutocompleteData() *model.AutocompleteData {
+	todo := model.NewAutocompleteData("todo", "[command]", "Avaliable commands: list, add")
+
+	add := model.NewAutocompleteData("add", "[message]", "Adds a Todo")
+	add.AddTextArgument("Add a todo", "[message]", "")
+	todo.AddCommand(add)
+
+	list := model.NewAutocompleteData("list", "[name]", "Lists your Todo issues")
+	items := []model.AutocompleteListItem{{
+		HelpText: "Received Todos",
+		Item:     "in",
+	}, {
+		HelpText: "Sent Todos",
+		Item:     "out",
+	}}
+	list.AddStaticListArgument("Lists your Todo issues", false, items)
+	todo.AddCommand(list)
+
+	pop := model.NewAutocompleteData("pop", "", "Removes the Todo issue at the top of the list")
+	todo.AddCommand(pop)
+
+	send := model.NewAutocompleteData("send", "[user] [todo]", "Sends a Todo to a specified user")
+	send.AddTextArgument("Whom to send", "[@awesomePerson]", "")
```

Comment:
While testing this I've noticed that after picking  a user from the `@` mention picker the autocomplete stays closed. Should I file a JIRA?

Reply:
This is a known issue and is run by UX as well as @aaronrothschild. If you press space autocomplete will continue suggesting. Fix will need a rework of the at_mention_provider as well as suggestions component.

Reply:
@iomodo Is there a JIRA that we can link here for reference?